web/src/components/projects/CreateProjectForm.tsx

```diff
@@ -436,81 +474,144 @@ export default function CreateProjectForm({ approvedStudies, handleProjectCreate
+                    const researcher = watch(`additionalApprovedResearchers.${index}`);
+                    const selectedEnvironment = environments.find((env) => env.id === selectedEnvironmentId);
+                    const availableRoles = selectedEnvironment ? getAvailableRoles(selectedEnvironment.name) : [];
+                    const availableRolesToAdd = availableRoles.filter((role) => !researcher?.roles?.includes(role));
+
+                    return (
+                      <div key={field.id} className={styles["item-wrapper"]}>
+                        <div className={styles["researcher-content"]}>
+                          <div>
+                            <label htmlFor={`researcher-${index}`} className={styles["field-label"]}>
+                              Username:
+                            </label>
+                            <Controller
+                              name={`additionalApprovedResearchers.${index}.username` as const}
+                              control={control}
+                              rules={{
+                                required: "Username is required",
+                                validate: {
+                                  isNotEmpty: (value) => {
+                                    if (!value || value.trim() === "") {
+                                      return "Username is required";
+                                    }
+                                    return true;
+                                  },
+                                  notEmailPart: (value) => {
+                                    if (value.includes("@")) {
+                                      return `Enter only the username part (without ${domainName})`;
+                                    }
+                                    return true;
+                                  },
+                                  isUnique: (value) => {
+                                    const allUsernames = getValues("additionalApprovedResearchers").map(
+                                      (user) => user.username
+                                    );
+                                    const duplicateCount = allUsernames.filter((username) => username === value).length;
+                                    return duplicateCount <= 1 || "Username has already been entered";
+                                  },
+                                },
+                              }}
+                              render={({ field: usernameField, fieldState }) => (
+                                <div className={styles["username-input-wrapper"]}>
+                                  <div>
+                                    <input
+                                      {...usernameField}
+                                      id={`researcher-${index}`}
+                                      type="text"
+                                      placeholder="Valid username"
```

Comment:
I don't think this is a useful placeholder. at UCL what my "username" is depends on the context. I vote 
```suggestion
                                      placeholder="ccaxyz"
```
but other suggestions welcome